config/initializers/devise.rb

```diff
@@ -244,7 +244,7 @@
   # Turn scoped views on. Before rendering "sessions/new", it will first check for
   # "users/sessions/new". It's turned off by default because it's slower if you
   # are using only default views.
-  config.scoped_views = true
+  config.scoped_views = false
```

Comment:
falseにするとviewのパスや翻訳キーも変わってくる（usersではなく、deviseのデフォルトが優先的に使われるはず）ので、もう一度viewを再生成した方がいいかもしれません。

Reply:
こちらの件ですが、もともとは

```
rails generate devise:views users
```

のようにしてviewを作成したはずです。
ですが、scoped_viewsは今回使わないので、

```
rails generate devise:views
```

でviewを再生成して、最初に作ったviewは削除する必要があると思いますー。

あと、deviseのデフォルトに近づけるならコメントアウトするのがいいかもしれません。
（`rails generate devise:install`を再実行するのもあり）

```suggestion
  # config.scoped_views = false
```